apps/picsa-apps/extension-app/src/app/app.component.ts

```diff
@@ -48,6 +50,9 @@ export class AppComponent implements OnInit, AfterViewInit {
     this.resourcesService.ready();
     // eagerly load monitoring service to sync form data
     this.monitoringService.ready();
+    this.ready.set(true);
+    // check for available updates
+    await this.appUpdateService.checkForUpdates();
```

Comment:
nit(non-blocking): even though the check itself is async (so it makes sense to await), you could also call without the await so that the rest of the app can continue to initialise independently of whether we wait for this service or not. 

Sometimes I like to just trigger a function and not wait for the result if happy to let it just happen in the background (no reason to block other functionality)